packages/ceramic-core/src/document.ts

```diff
@@ -27,6 +27,20 @@ export interface InitOpts {
 
 const deepCopy = (obj: any): any => JSON.parse(JSON.stringify(obj))
```

Comment:
Really useful function for cloning. Maybe we can put it in the common utility class?

Reply:
Yeah I think there are more of these useful functions that we can pull out to a utils module.